src/main/java/org/jenkinsci/plugins/pipeline/utility/steps/json/ReadJSONStep.java

```diff
@@ -23,23 +23,29 @@
  */
 package org.jenkinsci.plugins.pipeline.utility.steps.json;
 
+import hudson.Extension;
 import org.jenkinsci.plugins.pipeline.utility.steps.AbstractFileOrTextStep;
 import org.jenkinsci.plugins.pipeline.utility.steps.AbstractFileOrTextStepDescriptorImpl;
 import org.jenkinsci.plugins.workflow.steps.StepContext;
 import org.jenkinsci.plugins.workflow.steps.StepExecution;
 import org.kohsuke.stapler.DataBoundConstructor;
-
-import hudson.Extension;
+import org.kohsuke.stapler.DataBoundSetter;
 
 import javax.annotation.Nonnull;
-import java.io.Serializable;
 
 /**
  * Reads a JSON file from the workspace.
  *
  * @author Nikolas Falco
  */
 public class ReadJSONStep extends AbstractFileOrTextStep {
+    /**
+     * System property name. To be used with "true" to force this step to always return POJO objects instead of default
+     * net.sf.json.JSON objects.
+     */
+    public static final String RETURN_POJO_PROPERTY_NAME = ReadJSONStep.class.getName() + ".returnPojo";
```

Comment:
```suggestion
    public static final boolean RETURN_POJO_DEFAULT = Boolean.parseBoolean(System.getProperty(ReadJSONStep.class.getName() + ".returnPojo"));
```

Reply:
Change done in latest rebase opun master.